tmt/schemas/provision/hardware.yaml

```diff
@@ -97,6 +97,37 @@ definitions:
     # empty `cpu`.
     minProperties: 1
 
+  # HW requirements: single `device` item
+  device:
+    type: object
+
+    properties:
+      device-name:
+        type: string
+
+      device-id:
+        type: string
+
+      driver:
+        type: string
+
+      vendor-name:
+        type: string
+
+      vendor-id:
```

Comment:
`vendor-id` is not documented, and I suppose it's a "typo" and the proper name should be just `vendor`.

Reply:
The same seems to apply to `device-id`.

Reply:
And both `vendor` and `device` should allow both strings and integers, some users might forget about quotes, for example.

```yaml
        anyOf:
          - type: string
          - type: integer
```

Reply:
Thank you for @happz! Update PR